composer.json

```diff
@@ -11,23 +11,24 @@
     "type": "library",
     "license": "BSD-3",
     "homepage": "https://www.phalconphp.com",
-    "authors" : [
+    "authors": [
```

Comment:
Feel free to add yourself also

Reply:
Actually, real author is Mark Garrett. I propose replace this block by
```
    "authors": [
        {
            "name": "Mark Garrett",
            "email": "mark@moderndeveloperllc.com"
        },
        {
            "name": "Contributors",
            "homepage": "https://github.com/phalcon/vokuro/graphs/contributors"
        }
    ]
```